mne/channels/layout.py

```diff
@@ -882,13 +888,32 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
         dtype = [('type', 'S8'), ('name', 'S8'),
                  ('x', 'f8'), ('y', 'f8'), ('z', 'f8')]
         data = np.loadtxt(fname, dtype=dtype)
-        pos_ = data[data['type'].astype(np.str) == 'eeg']
-        pos = np.vstack((pos_['x'], pos_['y'], pos_['z'])).T
-        ch_names_ = pos_['name'].astype(np.str)
+        pos = np.vstack((data['x'], data['y'], data['z'])).T
+        ch_names_ = data['name'].astype(np.str)
     else:
         raise ValueError('Currently the "%s" template is not supported.' %
                          kind)
     selection = np.arange(len(pos))
+
+    if transform is True:
+        pos = apply_trans(als_ras_trans_mm, pos)
```

Comment:
The ALS-to-RAS part of this line is redundant, the neuromag-trans below does this already (ALS and RAS are both right hand systems).

What we do need to do is convert montages represented in mm to m, but can we safely assume that all montages are represented in mm? Especially if users can specify their own montages (i.e., digitized EEG points...) I this might not be a safe assumption.... We could either guess (if the largest distance between two points is bigger than 10 they're probably represented in mm, unless someone was inventive with the placement of the reference electrode...) or else we would probably have to add an additional function parameter?